src/commands/project.ts

```diff
@@ -137,6 +138,20 @@ export async function createProject(node: NodeBase | undefined, api?: AtelierAPI
         return;
       }
 
+      // Technically a project is a "document", so tell the server that we created it
+      try {
+        await new StudioActions().fireProjectUserAction(api, name, OtherStudioAction.CreatedNewDocument);
```

Comment:
Do we need two new StudioActions objects here or can we create one into a const and use it twice?